### PR TITLE
Add DAITA to `README.md`

### PR DESCRIPTION
**Pull request by MarkusPettersson98**
This PR adds some information about DAITA to the project `README.md`.

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/wireguard-go/7)

### CODE REVIEW

**Review by Serock3 — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/wireguard-go/7)* status: all files reviewed, 1 unresolved discussion (waiting on @MarkusPettersson98)

___
*[`README.md` line 59 at r1](https://reviewable.io/reviews/mullvad/wireguard-go/7#-O-1Vb1p5BFAR8kpXqqu:-O-1Vb1q1MJSFbThKA-9:b-meue5s) ([raw file](https://github.com/mullvad/wireguard-go/blob/fcc93949a7d664d527bdbf067faefd0cb199be83/README.md#L59)):*
> ```Markdown
> ### DAITA
> 
> [DAITA](https://mullvad.net/en/blog/introducing-defense-against-ai-guided-traffic-analysis-daita) is a Mullvad-specific addition to wireguard-go which integrates the [maybenot](https://github.com/maybenot-io/maybenot) framework for traffic analysis defenses. To build wireguard-go with DAITA you need to initialize the `maybenot` submodule.
> ```

Perhaps clarify that it is a nested submodule, or just provide the command `git submodule update --init --recursive wireguard-go-rs`

**Comment by Serock3** [comment]
Also, maybe we should add something to the very top of the README. Perhaps
```markdown
# Go Implementation of [WireGuard](https://www.wireguard.com/) - MullvadVPN fork

This is an implementation of WireGuard in Go. The fork adds support for [DAITA](#daita).
```

**Review by MarkusPettersson98 — COMMENTED** [review]
Fair! Adding this :blush:

> *[Reviewable](https://reviewable.io/reviews/mullvad/wireguard-go/7#-:-O-5EWSM6YCXZBut5xUh:bbnfwll)* status: all files reviewed, 1 unresolved discussion (waiting on @Serock3)

___
*[`README.md` line 59 at r1](https://reviewable.io/reviews/mullvad/wireguard-go/7#-O-1Vb1p5BFAR8kpXqqu:-O-5EApYBIuQ7ytHlwEG:b2d6qh) ([raw file](https://github.com/mullvad/wireguard-go/blob/fcc93949a7d664d527bdbf067faefd0cb199be83/README.md#L59)):*
<details><summary><i>Previously, Serock3 (Sebastian Holmin) wrote…</i></summary><blockquote>

Perhaps clarify that it is a nested submodule, or just provide the command `git submodule update --init --recursive wireguard-go-rs`
</blockquote></details>

The nested submodule structure does not apply to this repository, `maybenot` is the only submodule and can be initialized with the command that is provided just below the intro text :blush:

```
git submodule update --init
```

**Review by Serock3 — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/wireguard-go/7)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @MarkusPettersson98)

___
*[`README.md` line 59 at r1](https://reviewable.io/reviews/mullvad/wireguard-go/7#-O-1Vb1p5BFAR8kpXqqu:-O-AQOcXBulkuBjwF_H6:bp5zrfw) ([raw file](https://github.com/mullvad/wireguard-go/blob/fcc93949a7d664d527bdbf067faefd0cb199be83/README.md#L59)):*
<details><summary><i>Previously, MarkusPettersson98 (Markus Pettersson) wrote…</i></summary><blockquote>

The nested submodule structure does not apply to this repository, `maybenot` is the only submodule and can be initialized with the command that is provided just below the intro text :blush:

```
git submodule update --init
```
</blockquote></details>

Yup, I forgot about that :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Go Implementation of [WireGuard](https://www.wireguard.com/)
+# Go Implementation of [WireGuard](https://www.wireguard.com/) - MullvadVPN fork
```
Comment (dlon) [inline README.md]: Nit: MullvadVPN -> Mullvad VPN

**Review by MarkusPettersson98 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/wireguard-go/7)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @Serock3)

___
*[`README.md` line 1 at r2](https://reviewable.io/reviews/mullvad/wireguard-go/7#gh-1636080061:-O-BQFJl3PoSlj0Bsg6G:bb6un1j) ([raw file](https://github.com/mullvad/wireguard-go/blob/a2b6057a963a3d532903a97551185a45dcf2fad1/README.md#L1)):*
<details><summary><i>Previously, dlon (David Lönnhager) wrote…</i></summary><blockquote>

Nit: MullvadVPN -> Mullvad VPN
</blockquote></details>

Done

**Pull request merged**